user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc

```diff
@@ -0,0 +1,189 @@
+= Branching Fedora IoT
+
+Branched is the name given to a version of Fedora that has "branched" from the rolling Rawhide tree and will become the next stable Fedora release. This document will detail the steps for branching Fedora IoT from Rawhide. 
+
+To complete these steps and open a pull request you should only need a valid [Fedora account](https://docs.fedoraproject.org/en-US/fedora-accounts/user/).
+
+== Working with the pungi-iot repository
+
+Clone and fork the Fedora IoT Pungi repository to make changes to the configuration files used to build Fedora IoT.
+
+----
+git clone https://pagure.io/fedora-iot/pungi-iot.git
+----
+
+
+Create the new branch and push to the upstream repository. Change `$release` to the new numeric branch of Fedora.
+This is not done as a PR and should be pushed directly to the upstream repository. 
+
+----
+git checkout main; git pull; git checkout -b f$release; git push --set-upstream upstream f$release
+----
+
+Example, used for Fedora 40:
+
+----
+git checkout main; git pull; git checkout -b f40 ; git push --set-upstream upstream f40
+----
+
+=== Rawhide (main)
+
+On your local fork, create a new branch for changes to Rawhide (main).
+
+Update the signing key for Rawhide from the main fedora-pungi repo - https://pagure.io/pungi-fedora/commits/main.
+As of Fedora 40 branching you can find this [here](https://pagure.io/pungi-fedora/blob/main/f/fedora.conf=_21). 
```

Comment:
Looks like the URL got munged and dropped the `#`

Correct URL - https://pagure.io/pungi-fedora/blob/main/f/fedora.conf#_21

Reply:
thanks, that was my sed command